cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/UnmanagedClusterInventory.tsx

```diff
@@ -1,32 +1,72 @@
 // React imports
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
 // Library imports
 import { CdsButton } from '@cds/react/button';
 import { CdsIcon } from '@cds/react/icon';
+import { ClarityIcons, disconnectIcon } from '@cds/core/icon';
+import { CdsAlert, CdsAlertGroup } from '@cds/react/alert';
+
+// App imports
+import { NavRoutes } from '../../shared/constants/NavRoutes.constants';
+import { UnmanagedCluster } from '../../swagger-api';
+import { UnmanagedService } from '../../swagger-api/services/UnmanagedService';
+import UnmanagedClusterInfo from '../../shared/components/UnmanagedClusterInfo/UnmanagedClusterInfo';
+import './UnmanagedClusterInventory.scss';
+
+ClarityIcons.addIcons(disconnectIcon);
 
 const UnmanagedClusterInventory: React.FC = () => {
+    const [unmanagedClusters, setUnmanagedClusters] = useState<UnmanagedCluster[]>([]);
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        // fetch unmanaged clusters
+        UnmanagedService.getUnmanagedClusters().then((data) => setUnmanagedClusters(data));
+    }, []);
+
     return (
-        <div cds-layout="grid vertical col:12 gap:xl align:fill">
+        <div className="management-cluster-landing-container" cds-layout="grid vertical col:12 gap:lg align:fill">
             <div cds-layout="grid horizontal col:12">
-                <div cds-layout="vertical col:8 gap:lg">
+                <div cds-layout="vertical gap:md gap@md:lg col@sm:8 col:8">
                     <div cds-text="title">
-                        <CdsIcon cds-layout="m-r:sm" shape="block" size="xl" className="icon-blue"></CdsIcon>
-                        Unmanaged Cluster Inventory
+                        <CdsIcon cds-layout="m-r:sm" shape="computer" size="xl" className="icon-blue"></CdsIcon>
+                        Unmanaged Cluster
                     </div>
-                    <div cds-text="subsection" cds-layout="p-y:md">
-                        Placeholder page for unmanaged cluster inventory
+                    <div cds-text="subsection">
+                        Create a single node, local workstation cluster suitable for a development/test environment. It requires minimal
+                        local resources and is fast to deploy. It provides support for running multiple clusters. The default Tanzu
+                        Community Edition package repository is automatically installed when you deploy an unmanaged cluster.
+                    </div>
+                    <div cds-layout="vertical gap:md gap@md:lg col@sm:12 col:12">
+                        <CdsButton className="cluster-action-btn" status="primary" onClick={() => navigate(NavRoutes.DEPLOY_PROGRESS)}>
```

Comment:
This is fine to me for now, but am I right in expecting that in the future this would link to the unmanaged wizard?

Reply:
Yes this is for the wizard but the wizard is not done so I have not linked it.